bool grasp_has_plan
---
bool success
